internal/testing/setup/setup.go

```diff
@@ -181,23 +182,21 @@ func NewAzureTestPipeline(ctx context.Context, t *testing.T, accountName string,
 	}
 
 	httpClient = azureHTTPClient(r)
-	p := newPipeline(credential, r)
+	p := newPipeline(credential, api, r)
 
 	return p, done, httpClient
 }
 
-func newPipeline(c azblob.Credential, r *recorder.Recorder) pipeline.Pipeline {
+func newPipeline(c azblob.Credential, api string, r *recorder.Recorder) pipeline.Pipeline {
 	if c == nil {
 		panic("pipeline credential can't be nil")
 	}
 
 	f := []pipeline.Factory{
 		// sets User-Agent for recorder
 		azblob.NewTelemetryPolicyFactory(azblob.TelemetryOptions{
-			Value: "X-Az-Target",
+			Value: useragent.AzureUserAgentPrefix(api),
 		}),
-		// sets header X-Ms-Client-Request-Id, see https://msdn.microsoft.com/en-us/library/mt766820.aspx
```

Comment:
@vsaroopchand I wasn't sure why this header was needed, and it seems to work fine without it, so I removed it. If there's a reason for it, please let me know and I'll put it back with a comment explaining why.

Reply:
Its safe to remove otherwise any change to Version will require re-recording these tests. If you keep it, use a static value such as go-cloud-test, etc.